File: 37-Python/Tennis.py
from Campo import Campo
class Tennis(Campo):
    def __init__(self, codice, sport, nomeCampo, larghezza, lunghezza, costoOrario, temperatura, terreno) -> None:
        super().__init__(codice, sport, nomeCampo, larghezza, lunghezza, costoOrario)
        self._temperatura = temperatura
        self._terreno = terreno

    def __str__(self) -> str:
        return super().__str__() +"\t"+ str(self._temperatura) +"\t"+ self._terreno +"\t-\t-\t"+ str(self._costoOrario)

File: 37-Java/src/Tennis.java
public class Tennis extends Campo {
    private float temperatura;
    private String terreno;
    
    public Tennis(int codice, String sport, String nomeCampo, Float larghezza, Float lunghezza, Float costoOrario,
            float temperatura, String terreno) {
        super(codice, sport, nomeCampo, larghezza, lunghezza, costoOrario);
        this.temperatura = temperatura;
        this.terreno = terreno;
    }

    public String toString(){
        return super.toString() +"\t"+ temperatura +"\t"+ terreno +"\t-\t-\t"+ super.costoOrario;
    }
}
